compiler-core/src/build/project_compiler.rs

```diff
@@ -49,6 +49,7 @@ pub struct Options {
     pub target: Option<Target>,
     pub codegen: Codegen,
     pub warnings_as_errors: bool,
+    pub no_print_progress: bool,
```

Comment:
This is a command line interface, but the time this is run we already have a telemetry object, so this can be removed.